src/algebra/big_operators.lean

```diff
@@ -565,6 +565,23 @@ begin
   omega
 end
 
+lemma prod_Ico_reflect (f : ℕ → β) (k : ℕ) {m n : ℕ} (h : m ≤ n) :
+  ∏ j in Ico (n + 1 - m) (n + 1 - k), f j = ∏ j in Ico k m, f (n - j) :=
+begin
+  cases le_total k m with hkm hkm,
+  { rw [← finset.Ico.image_const_sub (le_trans hkm h)],
+    apply prod_image,
+    simp only [Ico.mem],
+    rintros i ⟨ki, im⟩ j ⟨kj, jm⟩ Hij,
+    rw [← nat.sub_sub_self (le_trans (le_of_lt im) h), Hij,
+      nat.sub_sub_self (le_trans (le_of_lt jm) h)] },
+  { simp [Ico.eq_empty_of_le, nat.sub_le_sub_left, hkm] }
+end
+
+lemma sum_Ico_reflect {δ : Type*} [add_comm_monoid δ] (f : ℕ → δ) (k : ℕ) {m n : ℕ} (h : m ≤ n) :
+  ∑ j in Ico (n + 1 - m) (n + 1 - k), f j = ∑ j in Ico k m, f (n - j) :=
+@prod_Ico_reflect (multiplicative δ) _ f k m n h
```

Comment:
Do you not want to add the `[to_additive]` tag?

Reply:
If not, maybe leave a short comment in the code, explaining why.

Reply:
`[to_additive]` will try to replace `1` with `0`.

Reply:
Sure, so it cannot prove the additive version automatically. But you can still register one as the additive version of the other, not?

Reply:
It will be unusable because the issue is at the level "convert the type", not at the level "convert the proof". `sum_Ico_reflect` doesn't have the type computed by `to_additive` based on `prod_Ico_reflect`.

A proper fix is to maintain a list of types (e.g., `nat`, `int`, `equiv.perm _`) that should be ignored when converting types/proofs but this is not easy.